sources/packages/web/src/services/EducationProgramOfferingService.ts

```diff
@@ -238,7 +185,7 @@ export class EducationProgramOfferingService {
     locationId: number,
     programId: number,
     offeringId: number,
-    payload: OfferingDTO,
+    payload: EducationProgramOfferingAPIInDTO,
   ): Promise<void> {
     return ApiClient.EducationProgramOffering.requestChange(
```

Comment:
await?

Reply:
Although not for this PR but I see a return on a void promise.

Reply:
Yes, technically we can return a void promise. And awaiting for that returned promise will wait until all the execution inside this method is complete.